zim.cc

```diff
@@ -290,7 +292,7 @@ bool ZimFile::open()
   for( quint32 i = 0; i < zimHeader.clusterCount; i++ )
     clusterOffsets[ i ] = QPair< quint64, quint32 >( offs.at( i ), i );
 
-  qSort( clusterOffsets );
+  std::sort(clusterOffsets.begin(), clusterOffsets.end());
```

Comment:
Please restore spaces after/before the parentheses here and in another place above.

Reply:
`std::sort( clusterOffsets.begin(), clusterOffsets.end() );`
What's wrong here?

Reply:
My comment ends with *and in another place above.*